build/FalkorDB-rs/Makefile.defs

```diff
@@ -0,0 +1,10 @@
+
+ifneq ($(SAN),)
+export FalkorDBRS=$(FalkorDBRS_BINDIR)/x86_64-unknown-linux-gnu/debug/libFalkorDB_rs.a
```

Comment:
isn't this path `/x86_64-unknown-linux-gnu/` is too specific ? what will happen when running SAN on ARM ?

Reply:
Currently cant run on arm